src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components/macro';
+import home from './Home.svg';
+
+function Dashboard({ letActiveUser }) {
```

Comment:
it's not a mistake, but I think using `let` at the start of naming isn't that good, because `let` is already used in JS

Reply:
Agreed
```suggestion
function Dashboard({ activeUser }) {
```